rootstrap/flutter-base
# Feature: manage modules globally

### Description
- Adding Melos: https://melos.invertase.dev/getting-started

### Instructions:
- Under the project root execute: `dart pub global activate melos 2.9.0` to install Melos globally.
- To test run: `melos doctor`
- Add the needed configurations command in the `melos.yml` file
- Check the [Melos documentation](https://melos.invertase.dev/getting-started) for more info

https://github.com/rootstrap/flutter-base/assets/17047904/0108f2f3-c4c0-489f-bede-2add02c7898b
